klog_test.go

```diff
@@ -1067,6 +1067,10 @@ func TestKvListFormat(t *testing.T) {
 			keysValues: []interface{}{"pod", "kubedns", "values", []string{"deployment", "svc", "configmap"}},
 			want:       " pod=\"kubedns\" values=[deployment svc configmap]",
 		},
+		{
+			keysValues: []interface{}{"pod", "kubedns", "[]byte", []byte("test for byte array")},
```

Comment:
Can you change the name of the key `[]byte`. It's doesn't match our naming conventions and it's easy to confuse with type. Just `bytes` should be good.